### ISSUE TITLE
Alt attribute

### ISSUE THREAD

**Issue by carodelafuente** [AngelikAn2ny/prj-rev-bwfs-dasmoto#1]
Try getting in the habit of using the `alt` attribute on your `<img>` tags. The alt attribute is the 'alternative text' used if the image needed to be replaced when a user cannot reach the image, whether it be for an error, or accessibility purposes. 

We actually have a great resource to get you to start thinking about accessibility: https://www.codecademy.com/articles/accessibility
